### ISSUE TITLE
Ignore @each for max-nesting-depth

### ISSUE THREAD

**Issue by dbartholomae** [bjankord/stylelint-config-sass-guidelines#33]
Hi there,

the `max-nesting-depth` rule currently doesn't ignore `@each` at the moment, which makes this code fail linting:
```
.button {
  // Button colors
  @each $key, $value in $colors {
    &__button--#{$key} {
      background-color: $value;
    }
  }
}
```
It would be great to modify the rule to allow for this kind of code as it is in line with the Sass Guidelines:
```
    "max-nesting-depth": [
      1,
      {
        "ignoreAtRules": [
          "media",
          "supports",
          "include",
          "each"
        ]
      }
    ]
```
If there's interest, I'll create the PR.

**Comment by bjankord** [comment]
I’m open to a PR to update the rule

**Comment by dbartholomae** [comment]
I've created a PR for updating the rule including updated tests. Furthermore I've added an unrelated PR to make tests run on windows.

**Comment by bjankord** [comment]
I've merged the windows PR and left a comment on the @each rule PR. Once that comment is resolved, I'll merge that PR and then update the changelog and cut a release. Thanks again @dbartholomae for the contribution!

**Comment by dbartholomae** [comment]
Done! :)

**Comment by bjankord** [comment]
This has been released in [v5.1.0](https://www.npmjs.com/package/stylelint-config-sass-guidelines)
